config/webhook/manifests.yaml

```diff
@@ -0,0 +1,54 @@
+
+---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: mutating-webhook-configuration
+webhooks:
+- clientConfig:
+    caBundle: Cg==
+    service:
+      name: webhook-service
+      namespace: system
+      path: /default-etc-improbable-io-v1-etcdclusters
+  failurePolicy: Fail
+  name: default.etcd.improbable.io
+  rules:
+  - apiGroups:
+    - etcd.improbable.io
+    apiVersions:
+    - v1
+    operations:
+    - CREATE
+    - UPDATE
+    resources:
+    - etcdclusters
+    - etcdpeers
+
+---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: validating-webhook-configuration
+webhooks:
+- clientConfig:
+    caBundle: Cg==
```

Comment:
This gets filled in by cert-manager ca injector: https://docs.cert-manager.io/en/latest/reference/cainjector.html